client/src/components/forms/moodForm/AttributeModal.js

```diff
@@ -27,6 +27,9 @@ const useStyles = makeStyles(theme => ({
 
 const AttributeModal = () => {
   const classes = useStyles();
+  const handleSubmit = () => {
```

Comment:
to prevent your form from submitting as a GET to /, you'll need to consume the event as a parameter to this function, and call `e.preventDefault()` see: https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault